app/frontend/src/submissions/components/SubmissionForm/Liner.vue

```diff
@@ -68,7 +68,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
             id="linerFrom"
             label="Liner From"
             hint="ft (bgl)"
-            type="number"
+            type="text"
             v-model.number="linerFromInput"
```

Comment:
does this work without throwing a console error when starting to type a decimal number like `.1` (without a leading 0)? If not we might need to remove the `.number` part of `v-model.number` as well.